docs/config-carbon.rst

```diff
@@ -104,6 +104,10 @@ Important notes before continuing:
 * ``aggregationMethod`` specifies the function used to aggregate values for the next retention level.  Legal methods are ``average``, ``sum``, ``min``, ``max``, and ``last``. The default is ``average``.
```

Comment:
Should you include percentiles' aggregation on this list too?

Reply:
My PR concerns carbon-aggregator aggregation methods, which I believe to be distinct from whisper storage aggregation methods that are defined [here](https://github.com/wridgers/whisper/blob/master/whisper.py#L552-L574).

Looks like the docs don't include `avg_zero`, `absmax`, or `absmin`.

I don't *think* it would make sense to add percentile aggregation to whisper file storage aggregation, what do you think?

Reply:
>My PR concerns carbon-aggregator aggregation methods, which I believe to be distinct from whisper storage aggregation methods that are defined here.

Yes, I got that - but then maybe the warning should be moved from line 107 to somewhere after line 176, i.e. after aggregator's aggregation methods?

Reply:
Ah yes, you are right. I understand now. Thinking back, I'm not sure why I decided 107 was a good place to put the warning! :)

Fixed and amended the offending commit!